src/routes/__layout.svelte

```diff
@@ -22,20 +21,40 @@
 	}
 
 	let siteId = import.meta.env.NODE_ENV === 'production' ? 2 : 1;
+	const scriptInjectString = `<script>
+		var _paq = (window._paq = window._paq || []);
+		/* tracker methods like "setCustomDimension" should be called before "trackPageView" */
+		_paq.push(['setDocumentTitle', document.domain + '/' + document.title]);
+		_paq.push(['setDomains', ['*.www.love-foundation.org', '*.love-foundation.org']]);
+		_paq.push(['trackPageView']);
+		_paq.push(['enableLinkTracking']);
+		(function () {
+			var u = '//matomo.love-foundation.org/';
+			_paq.push(['setTrackerUrl', u + 'matomo.php']);
+			_paq.push(['setSiteId', ${siteId}]);
+			var d = document,
+				g = d.createElement('script'),
+				s = d.getElementsByTagName('script')[0];
+			g.async = true;
+			g.src = u + 'matomo.js';
+			s.parentNode.insertBefore(g, s);
+		})();`;
+
+	const noscriptInjectString = `<noscript
+	><p>
+		<img
+			src="//matomo.love-foundation.org/matomo.php?idsite=${siteId}&amp;rec=1"
+			style="border:0;"
+			alt=""
+		/>
+	</p></noscript
+>`;
 </script>
 
 <svelte:head>
 	<!-- Matomo -->
-
-	<noscript
-		><p>
-			<img
-				src={`//matomo.love-foundation.org/matomo.php?idsite=${siteId}&amp;rec=1`}
-				style="border:0;"
-				alt=""
-			/>
-		</p></noscript
-	>
+	{@html scriptInjectString + '</script>'}
+	{@html noscriptInjectString}
```

Comment:
I had to do this this way to be able to programmatically set the siteId. SvelteHead can't interpret data at injection time otherwise :(